.github/workflows/unified-docker.yml

```
@@ -1,133 +1,133 @@
-          chmod +x docker/unified/build-image.sh
-          docker/unified/build-image.sh --${{ matrix.backend }}
-
-      - name: Push to GitHub Container Registry
-        if: ${{ !env.ACT }}
-        run: |
-          TAG="ghcr.io/mostlygeek/llama-swap:unified-${{ matrix.backend }}${{ matrix.variant.suffix }}"
-          docker push "${TAG}"
-          DATE_TAG=$(date -u +%Y-%m-%d)
-          docker tag "${TAG}" "${TAG}-${DATE_TAG}"
-          docker push "${TAG}-${DATE_TAG}"
+name: Build Unified Docker Image
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)